_topic_maps/_topic_map.yml

```
@@ -798,6 +798,13 @@ Topics:
 - Name: Creating quick start tutorials
   File: creating-quick-start-tutorials
   Distros: openshift-enterprise,openshift-origin
+- Name: Capabilities and layered products available for the OpenShift Container Platform web console
```

Update to a variation of 

```suggestion
- Name: Capabilities and layered products
```
Looks too long in the previews.